distmv/statdist.go

```diff
@@ -12,6 +12,14 @@ import (
 	"github.com/gonum/stat"
 )
 
+type DistNormaler interface {
```

Comment:
I can define them in my package, but here seemed like a natural place to define them. Why aren't they justifiable?